architecture/add-ons/index.md

```diff
@@ -156,8 +159,13 @@ The [Multi-Platform Controller] has no explicit resources of its own, but it rea
 
 ### Internal Services Controller
 
-The [Internal Services Controller] has a single resource, the `InternalRequest`, that is created in the managed namespace by the managed release pipeline run. An Internal Services Controller running on a different cluster watches for those and reconciles them to perform actions in another network zone. 
+The [Internal Services Controller] has a single resource, the `InternalRequest`, that is created in the managed namespace by the managed release pipeline run. An Internal Services Controller running on a different cluster watches for those and reconciles them to perform actions in another network zone.
+
+### MintMaker
+
+The [MintMaker] automates dependency updates for Konflux components using Renovate. It introduces the `DependencyUpdateCheck` custom resource that triggers dependency scanning across Components, creating Tekton PipelineRuns to execute Renovate scans and generate pull requests with dependency updates.
```

Comment:
I think that it's important to note that Mintmaker Pipelineruns run in the mintmaker namespace, thus the users can't see their status, logs etc...(which is a drawback IMO).